meteor/server/api/showStyles.ts

```diff
@@ -75,50 +75,94 @@ export async function insertShowStyleBaseInner(organizationId: OrganizationId |
 		_rundownVersionHash: '',
 	}
 	ShowStyleBases.insert(showStyleBase)
-	await insertShowStyleVariantInner(showStyleBase, 'Default')
+
+	const showStyleVariant: ShowStyleVariant = {
+		_id: getRandomId(),
+		name: 'Default',
+		blueprintConfigWithOverrides: wrapDefaultObject({}),
+		_rundownVersionHash: '',
+		_rank: 500,
+		showStyleBaseId: showStyleBase._id,
+	}
+	await insertShowStyleVariant(showStyleVariant)
 	return showStyleBase._id
 }
-export async function insertShowStyleVariant(
-	context: MethodContext | Credentials,
-	showStyleBaseId: ShowStyleBaseId,
-	name?: string
-): Promise<ShowStyleVariantId> {
+async function assertShowStyleBaseAccess(context: MethodContext | Credentials, showStyleBaseId: ShowStyleBaseId) {
 	check(showStyleBaseId, String)
 
 	const access = await ShowStyleContentWriteAccess.anyContent(context, showStyleBaseId)
 	const showStyleBase = access.showStyleBase
 	if (!showStyleBase) throw new Meteor.Error(404, `showStyleBase "${showStyleBaseId}" not found`)
-
-	return insertShowStyleVariantInner(showStyleBase, name)
 }
-export async function insertShowStyleVariantInner(
-	showStyleBase: ShowStyleBaseLight,
+
+export async function createDefaultShowStyleVariant(
```

Comment:
This might be personal opinion / nag, but what was wrong with the old name `insertShowStyleVariant`?

The new name differs much more from it's neighbor `insertShowStyleBase`, which gives me the impression that it does something considerably different, which it's not.

So the functionality of the function basically hasn't changed, so why change the name?

Reply:
I agree, I don't see the point in the name change, other than a collision with a private `insertShowStyleVariant` method, that didn't do anything beyond calling `ShowStyleVariants.insertAsync()`, which seems superfluous?